packages/dev-scripts/package.json

```diff
@@ -55,7 +55,7 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-config-salesforce": "^0.1.0",
     "eslint-config-salesforce-license": "^0.1.0",
-    "eslint-config-salesforce-typescript": "^0.2.1",
+    "eslint-config-salesforce-typescript": "^0.2.0",
```

Comment:
@shetzel I noticed that version 0.2.1 of this module does not exist, so pushing it down to 0.2.0.

```
npm show eslint-config-salesforce-typescript versions   
[ '0.1.0', '0.2.0' ]
```